nf_core/pipeline-template/conf/test.config

```diff
@@ -22,7 +22,7 @@ params {
     // Input data
     // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
     // TODO nf-core: Give any required params for the test so that command line flags are not needed
-    input  = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_test_illumina_amplicon.csv'
+    input  = ('https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_test_illumina_amplicon.csv', checkIfExists: true)
```

Comment:
Will this break normal `nextflow run main.nf -profile test,docker`?

Reply:
how about we set the `profile "docker"` in `nf-test.config` ? and the input is set here in `test.config` ?